### PR TITLE
Adding/Modified e2e tests for performance profiles

### LINKED ISSUE #12085: [E2E]: Compare results of Performance Test
### Current Behavior
Currently, Meshery’s E2E tests do not include comprehensive scenarios for comparing performance test results across different runs. This limits the ability to automatically verify comparison on performance test results feature

### Desired Behavior
1. User goto [Performance Profiles](https://playground.meshery.io/performance/profiles) pages.
2. User run multiple performance test in one performance profile.
3. User view the performance test results.
4. User selected multiple performance test result in one performance profile.
5. User click comparison button in top right corner in test profile.
6. User see the comparison modal open.

### Implementation
The test cases will be made under this [file](https://github.com/meshery/meshery/blob/master/ui/tests/e2e/service_mesh_performance.spec.js)

### Acceptance Tests
Develop E2E tests to verify that the UI correctly displays performance comparison results.

### Mockups

---
### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 🛠  [Contributing to Meshery's End-to-End Tests](https://docs.meshery.io/project/contributing/contributing-ui-tests)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by ayushsawant464**
**Notes for Reviewers**

This PR fixes #12085 

Tests included are:
1. User can add performance profile and run tests.
2. User can view detailed results.
3. User can edit the performance profile.
4. User can compare tests of a performance profile.
5. User can delete a performance profile.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by vishalvivekm** [comment]
@ayushsawant464 Thank you for your contributions! Let's discuss this during the Meshery Development call today at 9:00 AM CT | 7:30 PM IST. Adding it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit?pli=1#) || [meeting link](https://meet.layer5.io/meshery).

File: ui/tests/e2e/service-mesh-performance.spec.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+      await expect(page.getByText('Load test has been submitted')).toBeVisible();
+    });
+
+    test(`Compare test of a performance profile with load generator "${loadGenerator}" and service mesh "${serviceMesh}"`, async ({
+      page,
+    }) => {
+      await page.goto(`${ENV.MESHERY_SERVER_URL}/performance/profiles`);
+      await page.getByText(profileName, { exact: true }).first().click();
+      await page.getByRole('button', { name: 'View Results', exact: true }).first().click();
+      await page.locator('[data-testid="MUIDataTableBodyRow-0"] input[type="checkbox"]').check();
```
Comment (jerensl) [inline ui/tests/e2e/service-mesh-performance.spec.js:73]: Can we use playwright locator here `page.getByTestId`?

File: ui/tests/e2e/service-mesh-performance.spec.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+      await page.locator('[title="Compare selected"]').click();
+      await expect(page.locator('[id="chart-dialog-title"]')).toBeVisible();
+      await expect(page.getByRole('heading', { name: 'Comparison' })).toBeVisible();
```
Comment (jerensl) [inline ui/tests/e2e/service-mesh-performance.spec.js:77]: I think you can inject the test id in the modal and assert them here as well

File: ui/tests/e2e/service-mesh-performance.spec.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
+import { test, expect } from '@playwright/test';
+import { ENV } from './env';
+
+test.describe('Service Mesh Performance Management Tests', () => {
+  const peformanceProfiles = [
+    {
+      profileName: 'Sample-Perf-Test',
```
Comment (jerensl) [inline ui/tests/e2e/service-mesh-performance.spec.js:7]: can you add uuid after the name of profile name as well?

File: ui/tests/e2e/service-mesh-performance.spec.js [hunk 4]
```diff
@@ -0,0 +1,90 @@
+import { test, expect } from '@playwright/test';
+import { ENV } from './env';
+
+test.describe('Service Mesh Performance Management Tests', () => {
```
Comment (jerensl) [inline ui/tests/e2e/service-mesh-performance.spec.js:4]: Can we have `{ tag: '@unstable' }` here as well to make sure the test is stable first?

**Review by jerensl — CHANGES_REQUESTED** [review]

File: ui/tests/e2e/service-mesh-performance.spec.js (hunk 1 above)
Comment (ayushsawant464) [reply ui/tests/e2e/service-mesh-performance.spec.js:73]: @jerensl do u mean something like this 
await page.getByTestId('MUIDataTableBodyRow-0').locator('input[type="checkbox"]').check();
?
or should i inject testid ,i tried but couldnt find where to inject.

Comment (jerensl) [reply ui/tests/e2e/service-mesh-performance.spec.js:73]: @ayushsawant464 MUI test id is fine

**Review by jerensl — APPROVED** [review]

**Pull request merged**